.github/workflows/check-pr.yaml

```diff
@@ -7,6 +7,38 @@ on:
       - rel/*
 
 jobs:
+  check-commit-message:
+    name: Check commit message
+    runs-on: ubuntu-22.04
+    # Skip on draft, check on draft -> ready
+    if: github.repository_owner == 'Samsung' && github.event.pull_request.draft == false
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # Checkout PR head commit
+          # Checkout Action use merge commit as default
+          ref: ${{ github.event.pull_request.head.sha }}
+          # Fetch all history and branch (default: 1)
+          fetch-depth: 0
```

Comment:
Is this really needed?
```suggestion
```

Reply:
@mhs4670go Without that, github doesn't seem to pull the history log, thus it cannot get current commit message.